packages/store/test/unit/services/backend-system.test.ts

```diff
@@ -12,7 +12,7 @@ describe('BackendSystem service', () => {
     });
 
     beforeEach(() => {
-        jest.resetAllMocks();
+        // jest.resetAllMocks();
     });
```

Comment:
Remove it entirely?

Reply:
Done in 3d67970ea55934680e91cd61e1b191bd65053b33